src/hooks/useRuntime.ts

```diff
@@ -137,7 +137,7 @@ export function useRuntime(initial?: Parameters): IRuntimeContext {
       params:
         definition.params?.map((p: FxParamDefinition<FxParamType>) => ({
           ...p,
-          version: definition.version || "0",
```

Comment:
if we pass a default here, we will not be using the correct code to serialize params for the old snippet

Reply:
!!!!! you are the man that you identified this issue. jeah we really shouldn't set a "0" for default hahahaahh